backend/tournesol/admin.py

```diff
@@ -43,7 +39,18 @@ class VideoCriteriaScoreAdmin(admin.ModelAdmin):
 
 @admin.register(ContributorRating)
 class ContributorRatingAdmin(admin.ModelAdmin):
-    pass
+    list_display = (
+        'user',
+        'video',
+        'link_to_youtube',
+        'is_public',
+    )
+    list_filter = (
+        ('is_public', admin.AllValuesFieldListFilter),
```

Comment:
`admin.AllValuesFieldListFilter` is useful to build a list from the values present in the database . It was relevant for the `language` field in Video since the enumeration of all accepted languages would be very large.

As `is_public` is a boolean, no explicit filter class is required:
```suggestion
        'is_public',
```